client/views/room/MessageList/components/MessageReadReceipt.tsx

```diff
@@ -0,0 +1,27 @@
+import { css } from '@rocket.chat/css-in-js';
+import { Box, Icon } from '@rocket.chat/fuselage';
+import React, { ReactElement } from 'react';
+
+import { useSetting } from '../../../../contexts/SettingsContext';
+
+const MessageReadReceipt = (): ReactElement | null => {
+	const isReadReceiptEnabled = useSetting('Message_Read_Receipt_Enabled');
```

Comment:
Maybe on MessageListProvider? can we centralize the Settings there.

https://github.com/RocketChat/Rocket.Chat/blob/editContext/client/views/room/MessageList/providers/MessageListProvider.tsx#L26